### DISCUSSION TITLE
FastAPI and friends newsletter is broken

### DISCUSSION THREAD

**Discussion by strivingengineer** [fastapi/fastapi#9447] [Questions] [answered]
The link of https://fastapi.tiangolo.com/newsletter/ is broken and new person cannot subscribe to FastApi newsletter

<img width="1243" alt="Screen Shot 2023-04-27 at 6 50 53 AM" src="https://user-images.githubusercontent.com/113843456/234736251-d1d72626-d1ff-440c-8e15-fa5504740f68.png">

**Comment by hasansezertasan** [comment, marked as answer]
I just checked, it's working.

![Uploading image.png…]()
